packages/shared/src/components/comments/common.tsx

```diff
@@ -18,7 +17,8 @@ export function CommentPublishDate({
   );
 }
 
-export const commentBoxClassNames = `py-3 px-4 bg-theme-bg-secondary rounded-lg break-words typo-callout ${styles.commentBox}`;
```

Comment:
Did you drop the `commentBox` class on purpose?

Reply:
Yes, I had to apply it to a specific element to avoid unnecessary styling on some other parts but this should now be obsolete since the only content of this styling is the link-related we have discussed above.